Makefile

```
@@ -113,25 +127,43 @@ e2e-test:
 ##@ Build
 
 build: generate fmt vet ## Build manager binary.
-	go build -o bin/manager main.go
+	go build -o bin/manager -ldflags $(LDFLAGS) main.go
 
 run: manifests generate fmt vet ## Run a controller from your host.
 	go run ./main.go
 
 docker-build-manager: ## Build docker image with the manager.
-	docker buildx build $(_CACHE_FROM) $(_CACHE_TO) --platform="$(PLATFORM)" --output=$(OUTPUT_TYPE) --target manager -t ${MANAGER_IMG} .
+	docker buildx build \ 
+		$(_CACHE_FROM) $(_CACHE_TO) \
+		--build-arg LDFLAGS=$(LDFLAGS) \
+		--platform="$(PLATFORM)" \
+		--output=$(OUTPUT_TYPE) \
+		-t ${MANAGER_IMG} \
+		--target manager .
 
 docker-push-manager: ## Push docker image with the manager.
 	docker push ${MANAGER_IMG}
 
 docker-build-eraser:
-	docker buildx build $(_CACHE_FROM) $(_CACHE_TO) --platform="$(PLATFORM)" --output=$(OUTPUT_TYPE) -t ${ERASER_IMG} --target eraser .
+	docker buildx build \
```

```suggestion
	docker buildx build \
```

there's an extra space here, it's breaking ci